src/OpenTelemetry/CHANGELOG.md

```diff
@@ -24,6 +24,11 @@
 * Refactored temporality setting to align with the latest spec.
   ([#2666](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2666))
 
+* Removed the public properties `LongValue`, `DoubleValue`, `StartTime`, and
```

Comment:
Was thinking if something like this is more clear? Also fine as is, just a suggestion :)

> Removed the public properties `LongValue`, `DoubleValue`, `StartTime`, and
`EndTime` in favor of their counterpart public methods `GetLongSum`, `GetDoubleSum`,
`GetLastLongValue`, `GetLastDoubleValue`, `GetStartTime`, and `GetEndTime`.

Reply:
This sounds much better.